src/utils/cosmosdb_model.ts

```diff
@@ -212,6 +217,19 @@ export abstract class CosmosdbModel<
     )(newDocument, options);
   }
 
+  public batch(
+    // eslint-disable-next-line
+    operations: OperationInput[],
```

Comment:
Nice point, we have moved this method inside the `CosmosdbModelVersionedTTL` as `private`, it is only used by the `updateTTLForAllVersions` so it just accept `ReadonlyArray<PatchOperationInput>` at the moment, it can be extended in future.

Reply:
We can't narrow it down any more,` ReadonlyArray<PatchOperationInput>` it's the minimum.